swagger.json

```diff
@@ -5796,6 +5861,44 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "Contact": {
+            "type": "object",
+            "description": "Information about a contact.",
```

Comment:
Could we describe this as being a notification contact for alerts?

Reply:
Reworded the description.